pkg/healthcheck/healthcheck_multicluster.go

```diff
@@ -118,21 +118,21 @@ func (hc *HealthChecker) multiClusterCategory() category {
 			},
 			{
 				description: "service mirror controller has required permissions",
-				hintAnchor:  "l5d-multicluster-local-rbac-correct",
+				hintAnchor:  "l5d-multicluster-source-rbac-correct",
```

Comment:
Are changing these hint anchors going to require a website change as well?

Reply:
They probably will. I think a follow-up change in the website is okay for this though. I will do a scan through the tutorial when it is ready and make those changes after this is merged.

Reply:
There is a pending PR with all the website anchor changes that I will update once all of this lands. Have to incorporate the change that shows you the names of all alive/dead gateways that Alex did as well. So leave that with me :)